server/test/integration/enjoy_test.go

```diff
@@ -0,0 +1,43 @@
+package integration
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"server/graphql"
+	"testing"
+
+	"github.com/gin-gonic/gin/binding"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetEnjoyBadRequest(t *testing.T) {
+	// Create an HTTP request recorder
+	w := httptest.NewRecorder()
+
+	// Create an HTTP GET request
+	req, _ := http.NewRequest("GET", endpointEnjoy, http.NoBody)
+
+	// Execute the HTTP request
+	r.ServeHTTP(w, req)
+
+	// Check the HTTP status code is BadRequest
+	assert.Equal(t, http.StatusBadRequest, w.Code)
+}
+
+func TestGetEnjoyFromParis(t *testing.T) {
+	var resp graphql.GetEnjoyResponse
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", endpointEnjoy+"?City=Paris&Page=1", http.NoBody)
```

Comment:
You may need to change the query to small case

Reply:
Can you explain more what you want ?

Reply:
Change City => city
Page => page

Unit test may not pass if you don't chage this.